Makefile.toml

```diff
@@ -0,0 +1,98 @@
+[tasks.frontend]
+command = "./client/build"
+toolchain = "nightly"
+workspace = false
+dependencies = [ "atomify" ]
+
+[tasks.serve-frontend]
+script = [
+'''
+#!/usr/bin/env bash
+
+browser=$(command -v chromium 2>/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Please install chromium for the frontend as it's the only browser with offscreen canvas support!"
+  exit 1
+fi
+
+top=$(git rev-parse --show-toplevel 2>/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Please run this script from the ratatosk git repository!"
+  exit 1
+fi
+cd $top/client
+
+trap 'kill -TERM "$pid" 2>/dev/null' TERM INT
+python3 -m http.server 3000 &
+pid=$!
+chromium http://localhost:3000
```

Comment:
Not sure if it's sensible enough to name this chromium. IIRC the proprietary version calls its binary `google-chrome` which would break the script. An idea would be to look for multiple options, however this should live in its own file then.

Reply:
other possibilitys are:
```
google-chrome
google-chrome-dev
google-chrome-beta
chromium-snapshot-bin
```

Reply:
Or we could add a `I know what I'm doing` option, because it is used for dev purposes

Reply:
Done.